test/fixtures/templates/bad/resources/certificatemanager/domain_validation_options.yaml

```diff
@@ -16,3 +16,12 @@ Resources:
       DomainValidationOptions:
         - DomainName: "prefixed-example.com"
           ValidationDomain: "example.com"
+  Certificate3:
+    Type: AWS::CertificateManager::Certificate
+    Properties:
+      DomainName: "*.aws.domain.com"
+      ValidationMethod: DNS
+      DomainValidationOptions:
+        # Don't fail when one of the values don't exist
+        - DomainName: aws.domain.com
+          HostedZoneId: !ImportValue SubdomainHostedZoneId
```

Comment:
nitpick, but should this resource go here if it's valid? [`test/fixtures/templates/good/resources/certificatemanager/domain_validation_options.yaml`](https://github.com/aws-cloudformation/cfn-python-lint/blob/d5f7dce74f9dc714144f775c76e51547f730773f/test/fixtures/templates/good/resources/certificatemanager/domain_validation_options.yaml)